site/build-site.sh

```diff
@@ -29,7 +29,7 @@ fi
 cd homepage/
 if [ ! -d themes/hugo-universal-theme ]
 then
-  git clone --depth 1 --branch 1.1.1 https://github.com/devcows/hugo-universal-theme.git themes/hugo-universal-theme
+  git clone --depth 1 --branch 1.3.2 https://github.com/devcows/hugo-universal-theme.git themes/hugo-universal-theme
```

Comment:
Users having already done a maven build of the `site` module before this PR will get build errors here because an existing `themes/hugo-universal-theme` folder isn't getting updated.
How about checking the checked-out branch here?

Reply:
Yes, that is probably a good idea.